docs/resources/delegator-faq.md

```diff
@@ -19,7 +19,7 @@ Note that delegation are not necessarily bonded. Tokens of each delegator can be
 
 To delegate some IRIS token to a validator, you could run the following command:
 ```$xslt
-iriscli stake delegate  --address-delegator=<address-delegator> --address-validator=<address-validator> --chain-id=fuxi-3001 --from=name --gas=2000000 --fee=40000000000000000iris  --amount=10000000000000000000iris 
+iriscli stake delegate  --address-delegator=<address-delegator> --address-validator=<address-validator> --chain-id=irishub --from=name --gas=2000000 --fee=0.04iris  --amount=10000000000000000000iris 
```

Comment:
Gas and fee params given in the example won't work on Betanet, and the amount is too large.  This problem applies to all examples in this doc.

Reply:
all examples are set to `--fee=0.3iris --gas=50000` as you instructed before